### DISCUSSION TITLE
How can I report a warning as an error?

### DISCUSSION THREAD

**Discussion by santi100a** [remarkjs/remark#1217] [Q&A]
I'd like to report warnings from `remark-validate-links` as errors, to avoid using the `--frail` flag. Any help, please?

**Comment by wooorm** [comment]
Why not? That's what it's for?

**Reply by santi100a** [reply]
I think it is unnecessary to use `--frail`, and reporting an error as a "warning" is a little jarring. It'd be nice if Remark could be adjusted like ESLint to bypass the severity of certain rules.
